olive/passes/onnx/vitis_ai/quantize.py

```diff
@@ -166,6 +166,10 @@ def quantize_static(
     if not op_types_to_quantize or len(op_types_to_quantize) == 0:
         op_types_to_quantize = list(QLinearOpsRegistry.keys())
 
+    # not supported in ORT >= 1.16.0
+    # TODO: Update code to support different versions of ORT
+    from onnxruntime.quantization.quant_utils import load_model
+
     model = load_model(Path(model_input), optimize_model)
```

Comment:
@sheng-xiao , it seems the parameter is already removed in 1.16, should we remove it so that we could use load_model_with_shape_infer?

Reply:
This is the commit of interest FYI https://github.com/microsoft/onnxruntime/commit/89f8f20a61ceac0276c42d894c38400d86b175d0

`load_model_with_shape_infer` was changed to add a step `add_infer_metadata(model)` which was previously under `load_model`. If this is important, we might have to use `load_model` for versions < 1.16.0 and `load_model_with_shape_infer` for >=? 

same for `clone_model_with_shape_infer` which doesn't exist in 1.16.0

Reply:
Created an issue to track it https://github.com/microsoft/Olive/issues/382

Reply:
Thanks for your update on making the import lazily. We tested for the publicly released ORT 1.14/1.15 only. We are now working on updating Vitis-AI pass to be compatible with the nightly ORT version. Further discussion will be at https://github.com/microsoft/Olive/issues/382